lib/TopSitesFeed.jsm

```diff
@@ -375,12 +375,12 @@ this.TopSitesFeed = class TopSitesFeed {
 
     // Populate the state with available search shortcuts
     await new Promise(resolve => Services.search.init(resolve));
-    const searchShortcuts = SEARCH_SHORTCUTS.reduce((result, shortcut) => {
-      // Only add the shortcut if the engine is available.
-      if (!Services.search.getEngines().find(e => e.identifier && e.identifier.match(shortcut.searchIdentifier))) {
-        return result;
+    const searchShortcuts = Services.search.getDefaultEngines().reduce((result, engine) => {
+      if (!engine.identifier) { return result; }
```

Comment:
test failure. also, you can maybe just nest to avoid the early single line return

Reply:
```
        ✖ should dispatch UPDATE_SEARCH_SHORTCUTS on updateSearchShortcuts
          Firefox 54.0.0 (Linux 0.0.0)
        Services.search.getDefaultEngines is not a function
```